multi-tenancy-observability/README.md

```diff
@@ -0,0 +1,154 @@
+---
+creation_date: 2024-02-08
+issues:
+- https://github.com/giantswarm/roadmap/issues/2771
+owners:
+- https://github.com/orgs/giantswarm/teams/team-atlas
+state: review
+summary: How Giant Swarm manages multi-tenancy to isolate observability data per tenants (i.e. metrics, logs, traces).
+---
+
+# Multi-tenancy for observability
+
+## Glossary
+
+- `tenant`: a group of users who share a common access with specific privileges to observability data
+- `observability data`: all data related to metrics, logs, traces, profiles
```

Comment:
"data related to metrics [...]", or rather "metrics [...]" directly?

Reply:
```suggestion
- `observability data`: metrics, logs, traces, profiles and so on
```